chassis_control_topic_name: /control_cmd

cansend_topic_name: /sent_messages

node_rate: 50   # [Herz]

cansend_para:
  send_mode: 1 # 0 for autonomous driving; 1-test magnetic, 2-test steer
  test_steer_angle: -1
  test_acc_pedal: 0
  test_brk_pedal: 0
  setup_steer_speed: 300
  
  